src/sentry/models/group.py

```
@@ -45,6 +54,28 @@ class GroupStatus(object):
 class GroupManager(BaseManager):
     use_for_related_fields = True
 
+    def by_qualified_short_id(self, org, short_id):
+        match = _short_id_re.match(short_id.strip())
+        if match is None:
+            return
```

This seems like it should raise an assertion error?